samples/drivers/video/capture/Kconfig

```
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Video capture sample application"
+
+menu "Video capture configuration"
+
+config VIDEO_FORMAT_HEIGHT
+	int "Height of the video format"
+	default 0
+	help
+	  Height of the video format. If set to 0, the default height is used.
```

This is very oddly worded?


```suggestion
config VIDEO_FRAME_HEIGHT
	int "Height of the video frames"
	default 0
	help
	  Height of the video frames. If set to 0, the default height is used.
```

---

@kartben nice catch!